device/mellanox/x86_64-mlnx_msn4410-r0/plugins/psuutil.py

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/plugins/psuutil.py
```

Comment:
are you sure? i prefer in any case it will be using the sn4700 even for visuality